src/cleanvision/imagelab.py

```diff
@@ -21,29 +24,82 @@
 from cleanvision.utils.utils import get_filepaths, deep_update_dict
 from cleanvision.utils.viz_manager import VizManager
 
+__all__ = ["Imagelab"]
+
 OBJECT_FILENAME = "imagelab.pkl"
 TImagelab = TypeVar("TImagelab", bound="Imagelab")
 
 
 class Imagelab:
+    """A single class to find all types of issues in image datasets.
+
+    Parameters
+    ----------
+    data_path : str
+        Path to image files. Imagelab will recursively retrieve all image files from the specified path
+
+    Examples
+    --------
+
+    Basic usage of Imagelab class
+
+    .. code-block:: python
+        from cleanvision.imagelab import Imagelab
+        imagelab = Imagelab(data_path)
+        imagelab.find_issues()
+        imagelab.report()
+
+
+    Attributes
+    ----------
+    issues : pd.DataFrame
+        Dataframe containing scores on all issue types for all images in the dataset.
+        It has two types of columns for each issue type
+        1. <issue_type>_score - This column indicates the score of each image for that issue type. Lower scores imply
+        higher probability of issue_type being present in the image
+        2. <issue_type>_bool - This column indicates whether the issue_type is present in the image or not.
```

Comment:
```suggestion
    issues : pd.DataFrame
        Dataframe where each row corresponds to an image and columns specify which issues were detected in this image.
        It has two types of columns for each issue type:
        1. <issue_type>_score - This column contains a quality-score for each image for a particular type of issue. 
        Scores are between 0 and 1, lower values indicate images exhibiting more severe instances of this issue.
        2. <issue_type>_bool - This column indicates whether or not the issue_type is detected in each image (a binary decision rather than numeric score).
```